e2e/scripts/cleanup.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+kubectl delete postgresql acid-minimal-cluster
+kubectl delete deployments -l application=db-connection-pooler,cluster-name=acid-minimal-cluster
+kubectl delete statefulsets -l application=spilo,cluster-name=acid-minimal-cluster
+kubectl delete services -l application=spilo,cluster-name=acid-minimal-cluster
+kubectl delete configmap postgres-operator
+kubectl delete deployment postgres-operator
```

Comment:
This may be subject to a race condition where an operator is deleted earlier that it completes the clean up. That happened earlier both in this e2e pipeline and the `run_operator_locally` script.